projects/fbb-service/FellowshipOfTheCode/application.py

```diff
@@ -0,0 +1,42 @@
+from flask import Flask
+import json
+application = Flask(__name__)
+
+@application.route('/')
+def hello():
+    return "Hello there!"
+
+@application.route('/fizzbuzz/cap=<int:cap>&fizz=<fizz>&buzz=<buzz>&bazz=<bazz>')
```

Comment:
Is this route supposed to be:
/fizzbuzz?cap=...
vs
/fizzbuz/cap=

Reply:
As it currently stands, it is supposed to be '/fizzbuzz/cap...'.
I had troubles making it a query in the URL, so I made it a path and updated the yaml to reflect that.